app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job = Job.find(params[:job_id])
```

Comment:
i like that you are using the association to help assign the company's job, but to save a line, you could do the following:

```ruby
job = Job.find(params[:job_id])
@comment = job.comments.new(comment_params) 
```